conf/stats.conf.example

```diff
@@ -2,96 +2,105 @@
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_per_role
-mysql_query=select name, count(distinct node.mac) from node_category right join node on node.category_id=node_category.category_id where node.status='reg' group by node_category.category_id;
+mysql_query=select name, count(distinct node.mac) from node_category left join node on node.category_id=node_category.category_id and node.status='reg' group by node_category.category_id
 interval=60s
+randomize=true
 
 [metric 'total number of unregistered devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.unregistered
 mysql_query=select count(distinct mac) from node where status='unreg'
 interval=60s
+randomize=true
 
 [metric 'total online devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.online
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime is null
 interval=60s
+randomize=true
 
 [metric 'total offline devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.offline
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime <= NOW()
 interval=60s
+randomize=true
 
 [metric 'number of new registered devices during the past hour']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_last_hour
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 HOUR
-interval=60s
+interval=1s
```

Comment:
1 second seems to be pretty low to execute that query

Reply:
Innodb caching is pretty good (hot query < 1ms), and w/ my testing a cold query took about 3ms on `prod-pf`, so I don't expect cold cache to take much longer on higher-use hosts.

Reply:
prod-pf is pretty small though, I'm worry about environments with a lot of nodes

Anyway, we can address it after the release